### PR TITLE
Directive working in iPhone simulator, test added

### LINKED ISSUE #548: iOS double tap still an issue
As mentioned in #517 iOS browsers require multiple taps to save records. Fix presented in #544 seems to now also effect hyperlinks

### PR DESCRIPTION
**Pull request by susanley**
Closes #548.

### CODE REVIEW

**Comment by jfranco-gri** [comment]
I pulled this down and tried it on an iPhone simulator, but I'm still seeing the double-tap issue. The first time you click the "save record" button, you only have to click it once for it to be saved. But if you try it again with a different record, you have to click on the button twice for it to be saved.

**Comment by susanley** [comment]
@jfranco-gri Thanks! I'll try to duplicate the behavior in my simulator.

File: test/e2e/search-results.e2e.spec.js [hunk 1]
```diff
@@ -37,14 +37,32 @@ describe('Search Results', function() {
...
+      browser.switchTo().window(newWindowHandle).then(function () {
+        var windowWidth = newWindowHandle.innerWidth;
+        if (windowWidth < 960) {        
+          expect(browser.getCurrentUrl()).toContain('https://archive.org/details/handbookofarmsar00metr_0');
+          browser.ignoreSynchronization = false;
+        }
+      });
+    });
+    browser.ignoreSynchronization = false;
+  });
```
Comment (jfranco-gri) [inline test/e2e/search-results.e2e.spec.js]: For this test, we might want to check saving multiple records at once. Maybe have it go to the results page and have it save a few records (maybe 3?). Then have it check each record to see if it's 'saved.'

Also, rather checking the url, I would check that the bookmark icon is in the "saved" state (it should have the 'saved' class).  You can look at the below test, 'it should save records' for an example of testing that.  We can just check that by looking at the bookmark icon on either the results page or the item detail page.  Or we can look at the saved records page to make sure the correct saved records are displayed.

Comment (susanley) [inline test/e2e/search-results.e2e.spec.js]: Noted!

**Comment by susanley** [comment]
@joshuago78 @jfranco-gri Update: The new directive works in the simulator, but now seemingly unrelated tests fail when run through protractor, but seem fine when I test them manually. Investigating.

**Pull request closed without merging**